src/main/java/com/google/cloud/genomics/dataflow/utils/GenomicsOptions.java

```diff
@@ -33,28 +29,30 @@
  * annotated with a @JsonIgnore annotation.
  */
 public interface GenomicsOptions extends DataflowPipelineOptions {
+  
+  // Use the same appName for all pipelines in this collection so that the oauth flow only needs to happen once.
+  public static final String APP_NAME = "dataflow-java";
```

Comment:
OK, happy end then!

On Wed, Jul 1, 2015 at 1:36 PM, Nicole Deflaux <notifications@github.com>
wrote:

> In
> src/main/java/com/google/cloud/genomics/dataflow/utils/GenomicsOptions.java
> <https://github.com/googlegenomics/dataflow-java/pull/104#discussion_r33721295>
> :
>
> > @@ -33,28 +29,30 @@
> >   * annotated with a @JsonIgnore annotation.
> >   */
> >  public interface GenomicsOptions extends DataflowPipelineOptions {
> > +
> > +  // Use the same appName for all pipelines in this collection so that the oauth flow only needs to happen once.
> > +  public static final String APP_NAME = "dataflow-java";
>
> You are right, but in addition the change was not needed anyway since
> Dataflow's storage directory for the StoredCredential is the same for all
> pipelines unless the user changes it via --credentialDir. (Before when we
> were doing our own oauth flow, it was triggered for each different pipeline
> since appname was used in the directory path for the stored credential.)
>
> I reverted that change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlegenomics/dataflow-java/pull/104/files#r33721295>
> .
>